### PR TITLE
Update office-365-groups.md

### PR DESCRIPTION
**Pull request by jameseccles**
Added additional roles to the note on permissions

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/activity-reports/office-365-groups.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ description: "Get an Office 365 groups report to know about the groups and their
 The Office 365 **Reports** dashboard shows you the activity overview across the products in your organization. It enables you to drill in to individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md). In the Office 365 groups report, you can gain insights into the activity of groups in your organization and see how many groups are being created and used.
   
 > [!NOTE]
-> You must be a global administrator in Office 365 or an Exchange, SharePoint, or Skype for Business administrator to see reports. 
+> You must be a global administrator, global reader or reports reader in Office 365 or an Exchange, SharePoint, or Skype for Business administrator to see reports.
```
Comment (JohanFreelancer9) [inline Office365-Admin/activity-reports/office-365-groups.md]:
```suggestion
> You must be a global administrator, global reader, or reports reader in Office 365 or an Exchange, SharePoint, or Skype for Business administrator to see reports. 
```

**Comment by mypil** [comment]
@jameseccles - please merge the commit on the copy/edit review done by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. There's just a very minor copy/edit changes that wasn't merged by the requester but please double check if you can merge this. 

Thank you.

cc: @kenwith

**Pull request merged**